### PR TITLE
[0.3.0] Club Data Models

### PR DESCRIPTION
**Pull request by Galax028**
I have implemented the necessary data models for club functionality. This resolves BACK-65.

> [!NOTE]
> The data model is not queryable nor sortable yet, but the boilerplate is already in place in case someone wants to start working on that. It's good for now, but we'll add it if needed.

### CODE REVIEW

**Review by smartwhatt — CHANGES_REQUESTED** [review]
Few changes

File: mysk-lib/src/models/club/db.rs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+};
+use chrono::{DateTime, Utc};
+use mysk_lib_derives::{BaseQuery, GetById};
+use mysk_lib_macros::traits::db::{BaseQuery, GetById};
+use serde::Deserialize;
+use sqlx::{query, FromRow, PgPool, Postgres, QueryBuilder};
+use uuid::Uuid;
+
+#[derive(BaseQuery, Clone, Debug, Deserialize, FromRow, GetById)]
+#[base_query(query = "SELECT * FROM organizations")]
```
[the remark this answers is not in the document]
Comment (Galax028) [reply mysk-lib/src/models/club/db.rs:25]: My bad, I was copying much boilerplate around :skull:

**Review by smartwhatt — APPROVED** [review]
LGTM!!

**Review by owo93 — APPROVED** [review]
LGTM 😦

**Pull request merged**